spk/sabnzbd-testing/Makefile

```diff
@@ -15,7 +15,34 @@ DESCRIPTION_SPN = SABnzbd hace que Usenet sea lo más simple posible, automatiza
 ADMIN_PORT = 8089
 RELOAD_UI = yes
 DISPLAY_NAME = SABnzbd Testing
-CHANGELOG = "Update to 1.0.0"
+CHANGELOG = "1. Update to 1.0.0<br>2. Warning: When you upgrade an existing 0.7.x installation, release 1.0.0 will NOT be able to finish the queue. It's best to empty your queue first using 0.7.x. Alternatively, you can install 1.0.0 and then use queue repair on the Status page. Some information (like the order of the queue) will be lost.
```

Comment:
As per the comment in the other PR: this should be updated to say `Update to 1.0.3`. Let's leave the warning in for now.
The rest of the additions should be removed: Package Center won't be able to handle it.